legal-api/tests/unit/services/filings/validations/change_of_director/test_validation_effective_date.py

```diff
@@ -35,11 +26,14 @@
 
 def common_setup(identifier: str, now: datetime):
     """Set up the data for COD tests."""
+    founding_date = now - datedelta.YEAR
     filing_date = now - datedelta.DAY
 
+    # create business founded "a year ago"
```

Comment:
comment was captured in previous code??

Reply:
There was no comment in the previous code...?

Reply:
doesn't need a comment
founding_date=now - datedelta.YEAR
but now I have to read the comment or scroll up to look at the data to know that ... and if the variable assignment changes, then so does this (not close in proximity) comment. so less cognitively clean in my mind.
again, sort of a format style change without any real effects.

Reply:
Comment reverted in next commit.